core/src/main/java/bio/terra/pearl/core/service/logging/MixpanelService.java

```diff
@@ -69,13 +69,13 @@ public void logEvent(String data) {
     }
 
     protected JSONObject buildEvent(JSONObject event) {
-        MessageBuilder messageBuilder = new MessageBuilder(env.getProperty("mixpanel.token"));
+        MessageBuilder messageBuilder = new MessageBuilder(env.getProperty("env.mixpanel.token"));
```

Comment:
while you're doing this, might as well pull it into a constant since we have to use it twice